packages/discovery-provider/src/api/v1/users.py

```diff
@@ -2396,3 +2397,101 @@ def get(self, id, authed_user_id):
         response = Response(withdrawals, content_type="text/csv")
         response.headers["Content-Disposition"] = "attachment; filename=withdrawals.csv"
         return response
+
+
+remixers_parser = pagination_with_current_user_parser.copy()
+remixers_parser.add_argument(
+    "track_id",
+    required=False,
+    description="Filters for remixers who have remixed the given track ID",
+    type=str,
+)
+remixers_reponse = make_response(
+    "remixers_response", ns, fields.List(fields.Nested(user_model))
+)
+full_remixers_reponse = make_full_response(
+    "full_remixers_response", full_ns, fields.List(fields.Nested(user_model_full))
+)
+remixers_count_response = make_full_response(
+    "purchases_count_response", full_ns, fields.Integer()
+)
+
+USER_REMIXERS_ROUTE = "/<string:id>/remixers"
+
+
+@full_ns.route(USER_REMIXERS_ROUTE)
+class FullRemixersUsers(Resource):
+    @log_duration(logger)
+    def _get_user_remixers(self, id):
+        decoded_id = decode_with_abort(id, full_ns)
+        args = remixers_parser.parse_args()
+        limit = get_default_max(args.get("limit"), 10, 100)
+        offset = get_default_max(args.get("offset"), 0)
+        current_user_id = get_current_user_id(args)
+        track_id = args.get("track_id")
+        decoded_track_id = decode_with_abort(track_id, full_ns) if track_id else None
+        args = GetRemixersArgs(
+            remixee_user_id=decoded_id,
+            current_user_id=current_user_id,
+            track_id=decoded_track_id,
+            limit=limit,
+            offset=offset,
+        )
+        remixers = get_remixers(args)
+        users = list(map(extend_user, remixers))
+        return success_response(users)
+
+    @full_ns.doc(
+        id="""Get remixers""",
+        description="Gets the list of unique users who have remixed tracks by the given user, or a specific track by that user if provided",
+        params={"id": "A User ID"},
+        responses={200: "Success", 400: "Bad request", 500: "Server error"},
+    )
+    @full_ns.expect(pagination_with_current_user_parser)
```

Comment:
Wrong parser here too... the `track_id` query param is missing from the generated output

Reply:
whoops good catch thx